backend/src/db/migrations/20230822132525_create_articles_table.ts

```diff
@@ -28,7 +28,7 @@ const up = (knex: Knex): Promise<void> => {
       .references(ColumnName.ID)
       .inTable(TableName.USERS);
     table.integer(ColumnName.PROMPT_ID).unsigned().nullable();
-    table.integer(ColumnName.GENRE_ID).unsigned();
+    table.integer(ColumnName.GENRE_ID).unsigned().nullable();
```

Comment:
Create new migration to alter table with nullable column
It won't be applied on deployed env as this migration has been already been run no deployed app database
It's almost always a bad idea to edit old migrations 🥲 
```suggestion
    table.integer(ColumnName.GENRE_ID).unsigned();
```

Reply:
Got it, thanks!

Reply:
maybe it isn't necessary to create a new migration because it is nullable by default and does not change anything?

Reply:
@IrynaK-a I guess it's just for code consistency

Reply:
Yeah, I've missed thing with default nullable 😅
Just recognized changed migration - let's not create new migration (but remove changes from existing one)